packages/jest-cli/src/reporters/SummaryReporter.js

```diff
@@ -77,17 +84,16 @@ class SummareReporter extends BaseReporter {
   onRunStart(
     config: Config,
     aggregatedResults: AggregatedResult,
-    runnerContext: RunnerContext,
     options: ReporterOnStartOptions,
   ) {
-    super.onRunStart(config, aggregatedResults, runnerContext, options);
+    super.onRunStart(config, aggregatedResults, options);
     this._estimatedTime = options.estimatedTime;
   }
 
   onRunComplete(
+    contexts: Set<Context>,
     config: Config,
```

Comment:
is this going to be a config for project where the last test was run from?
this looks a little strange, since `onRunComplete` seems to be responsible for summarizing the whole (multi-project) run and not having any dependencies on a specific project